db/seeds.rb

```diff
@@ -5,3 +5,16 @@
 #
```

Comment:
When your application is deployed, should we not have some default admin set up?

Reply:
Yes Sir, I will add it.